sympy/printing/pretty/pretty.py

```diff
@@ -2067,7 +2067,7 @@ def _print_nth_root(self, base, root):
     def _print_Pow(self, power):
         from sympy.simplify.simplify import fraction
         b, e = power.as_base_exp()
-        if power.is_commutative:
+        if power.is_commutative is not False:
```

Comment:
The `is_commutative` attribute should always be True or False so it shouldn't be necessary to change this.

Reply:
Done !